### DISCUSSION TITLE
How to use terminal colours instead of a preset theme?

### DISCUSSION THREAD

**Discussion by thisisluxion** [LazyVim/LazyVim#1933] [Q&A]
I'm trying to use my default 16 terminal colours (I'm using kitty as my terminal emulator) as I have hand-picked them myself instead of using a premade theme. I tried setting the colorscheme to default but it won't work, and `colorscheme = function() end` won't work either. Is it possible to do so?

**Comment by pnitijarasrat** [comment]
-- In options.lua
```
vim.cmd.colorscheme = nil
vim.opt.termguicolors = false
vim.cmd("set t_Co=256")
```

-- In /plugin/color.lua
```
return {
    {
        "LazyVim/LazyVim",
        opts = {
            colorscheme = function() end, -- Disable LazyVim's colorscheme
        },
    },
}
```
